### ISSUE TITLE
Wrong error message

### ISSUE THREAD

**Issue by enumag** [phpstan/phpstan-symfony#18]
I encountered this error:

```
In XmlServiceMapFactory.php line 26:
                                                                                                                                          
  [PHPStan\Symfony\XmlContainerNotExistsException]                                                                                        
  Container .../vendor/phpstan/phpstan/../../../app/cache/dev/appDevDebugProjectContainer.xml not exists
```

This is however not true, the file does exist. I removed the `@` operator in XmlServiceMapFactory and got this:

```
PHP Warning:  simplexml_load_file(): .../vendor/phpstan/phpstan/../../../app/cache/dev/appDevDebugProjectContainer.xml:235: parser error : PCDATA invalid Char value 27 in .../vendor/phpstan/phpstan-symfony/src/Symfony/XmlServiceMapFactory.ph</parameter> in .../vendor/phpstan/phpstan-symfony/src/Symfony/XmlServiceMapFactory.php on line 23
```

**Comment by enumag** [comment]
The reason was an invalid character in one of my parameters causing the XML to be invalid. After that, it works fine. You might want to change the error message though.
